From 3a37ec47ffc9e4d34ac726d649a822cdead1b38f Mon Sep 17 00:00:00 2001
From: Jussi Kukkonen <jussi.kukkonen@intel.com>
Date: Wed, 30 Sep 2015 11:25:08 +0300
Subject: [PATCH] Apply xuser-specific policies if present

system-xuser.conf is installed by xuser-account and contains
policies that override the default service policies (allowing
xuser to send messages to the services).

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Jussi Kukkonen <jussi.kukkonen@intel.com>
---
 bus/system.conf.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/bus/system.conf.in b/bus/system.conf.in
index 851b9e6..1822011 100644
--- a/bus/system.conf.in
+++ b/bus/system.conf.in
@@ -82,6 +82,10 @@
        holes in the above policy for specific services. -->
   <includedir>system.d</includedir>
 
+  <!-- Apply xuser policies (if present) after the service
+       policies so the xuser ones don't get overridden. -->
+  <include ignore_missing="yes">system-xuser.conf</include>
+
   <!-- This is included last so local configuration can override what's 
        in this standard file -->
   <include ignore_missing="yes">system-local.conf</include>
-- 
2.1.4

